client/relay-chain-interface/src/lib.rs

```diff
@@ -105,6 +105,9 @@ pub trait RelayChainInterface: Send + Sync {
 	/// Get the hash of the current best block.
 	async fn best_block_hash(&self) -> RelayChainResult<PHash>;
 
+	/// Get the hash of the current best finalised block.
+	async fn last_finalised_block_hash(&self) -> RelayChainResult<PHash>;
```

Comment:
```suggestion
	async fn finalized_block_hash(&self) -> RelayChainResult<PHash>;
```

We use US English :)